server/graphql/v2/mutation/SendSurveyResponseMutation.ts

```diff
@@ -53,6 +53,10 @@ export const sendSurveyResponseMutation = {
     };
 
     try {
+      if (config.env === 'development') {
+        return true;
+      }
+
```

Comment:
This is a blocker to me: we need the ability to work on this feature in local dev environments. Shared my concerns in https://github.com/opencollective/opencollective/issues/7048#issuecomment-1876453672.